opensearch-operator/pkg/builders/cluster.go

```diff
@@ -971,3 +973,81 @@ func DataNodesCount(ctx context.Context, k8sClient client.Client, cr *opsterv1.O
 	}
 	return count
 }
+
+func NewServiceMonitor(cr *opsterv1.OpenSearchCluster) *monitoring.ServiceMonitor {
+	ClusterLabels := "opster.io/opensearch-cluster"
```

Comment:
This is already declared in `constants.go`, use it from there as `helpers.ClusterLabel` instead of redeclaring it here.

Reply:
Hey :) when i used the const it didn’t managed to use it, so i declared it again here

Reply:
Please merge in the current main branch then the constant will be available.